package.json

```
@@ -17,7 +17,7 @@
     "prepublishOnly": "cd scripts && ts-node filter-shrinkwrap.ts \"$(lerna list --parseable --scope ganache)\"/npm-shrinkwrap.json",
     "postpublish": "git restore \"$(lerna list --parseable --scope ganache)\"/npm-shrinkwrap.json",
     "start": "lerna exec --loglevel=silent --scope ganache -- npm run start --silent -- ",
-    "test": "lerna exec --concurrency 1 -- npm run test",
+    "test": "lerna exec --concurrency 1 --scope @ganache/version-check -- npm run test",
```

```suggestion
    "test": "lerna exec --concurrency 1 -- npm run test",
```
I assume this was just intended for while you were testing?

---

If so, one way you can set the scope in your CLI _before_ running `npm run test` (so that you don't actually have to alter the package.json like you did here is:
`$(npm bin)/lerna exec --scope @ganache/version-check -- npm t`